tests/api/test_bulk_delete.py

```diff
@@ -286,6 +286,46 @@ def test_delete_draft_orders(
         id__in=[order.id for order in orders]).count() == len(orders)
 
 
+def test_delete_draft_orders_lines(
+        staff_api_client, order_with_lines, permission_manage_orders):
+    order = order_with_lines
+    order_lines = [line for line in order]
+    order.status = OrderStatus.CANCELED
+
+    query = """
+    mutation draftOrderLinesBulkDelete($ids: [ID]!) {
+        draftOrderLinesBulkDelete(ids: $ids) {
+            count
+            errors {
+                field
+                message
+            }
+        }
+    }
+    """
+
+    variables = {'ids': [
+        graphene.Node.to_global_id('OrderLine', order_line.id)
+        for order_line in order_lines]}
+    response = staff_api_client.post_graphql(
+        query, variables,
+        permissions=[permission_manage_orders])
+    content = get_graphql_content(response)
+    assert 'errors' in content['data']['draftOrderLinesBulkDelete']
+    assert content['data']['draftOrderLinesBulkDelete']['count'] == 0
+
+    order.status = OrderStatus.DRAFT
```

Comment:
I think that part of this unittest, starting from here should be in another one. 
You are testing two different things. 1. The behaviour for non-draft orders 2. The behaviour for draft orders.

Reply:
Yep, agree. I just wanted to test draft and non draft order cases in one test, need to divide in two.